src/App.js

```diff
@@ -116,9 +171,81 @@ class App extends Component {
     this.setState({people: peopleList})
   }
 
-  onConfirm (e) {
+  onConfirmNames (e) {
     if (e.target.value.toLowerCase() === Y) {
-      this.setState({game: {gameState: PLAYING}})
+      const gameObject = Object.assign({}, this.state.game)
+      gameObject['gameState'] = PACKING
+      this.setState({game: gameObject})
+    }
+  }
+
+  onPackingChoice (e) {
+    const userChoice = parseInt(e.target.value, 10)
+    if ((e.keyCode === RETURN || e.keyCode === SPACEBAR) && [1, 2, 3, 4, 5, 6, 7].includes(userChoice)) {
+      const gameObject = Object.assign({}, this.state.game)
+      gameObject['gameState'] = ITEM
+      const inventoryChange = Object.assign({}, this.state.inventory)
+      const itemInventoryKey = packingMenu[userChoice - 1]
+      inventoryChange[itemInventoryKey] = CHANGING
+      this.setState({
+        game: gameObject,
+        inventory: inventoryChange
+      })
+    }
+  }
+
+  itemChanging (inventoryObject) {
+    for (let item in inventoryObject) {
+      if (inventoryObject[item] === CHANGING) {
+        return 'How many ' + itemRepresentation[packingMenu.indexOf(item)] + ' do you want to pack?'
+      }
+    }
+  }
+
+  limitPacking (inventoryItem) {
+    const difficultyLevel = this.state.game.difficulty
+    const itemLimit = difficultyLevels[difficultyLevel][inventoryItem]
+    return itemLimit
+  }
+
+  handleNumberToPack (e) {
+    let itemToChange
+    let message
+    if (e.keyCode === RETURN || e.keyCode === SPACEBAR) {
+      const userChoice = parseInt(e.target.value, 10)
+      const changingInventory = this.state.inventory
+      for (let item in changingInventory) {
+        if (changingInventory[item] === CHANGING) {
+          itemToChange = item
+        }
+      }
+      const itemLimit = this.limitPacking(itemToChange)
+      const changeToMake = itemLimit < userChoice ? itemLimit : userChoice
+      const userSuccess = itemLimit >= userChoice
+      message = this.packChangeRepresentation(userSuccess, changeToMake, itemToChange)
```

Comment:
You have a lot of methods that have to do with packing and state management in app. You *might* consider having a pure JavaScript file called `Pack.js` (for example) which is *not* a React component, but abstracts away some of these methods.